lib/chef/cookbook/metadata.rb

```diff
@@ -535,6 +535,25 @@ def source_url(arg = nil)
         )
       end
 
+      # This method translates version constraint strings from
+      # cookbooks with the old format.
+      #
+      # Before we began respecting version constraints, we allowed
+      # multiple constraints to be placed on cookbooks, as well as the
+      # << and >> operators, which are now just < and >. For
+      # specifications with more than one constraint, we return an
+      # empty array (otherwise, we're silently abiding only part of
+      # the contract they have specified to us). If there is only one
+      # constraint, we are replacing the old << and >> with the new <
+      # and >.
+      def handle_incorrect_constraints(specification)
+        specification.inject(Mash.new) do |acc, (cb, constraints)|
+          constraints = Array(constraints)
```

Comment:
Is there a reason to leave the commented out part here?

Reply:
Not at all.
Thanks for the spot, I'll tidy that up tomorrow